app/routes/events/components/CalendarCell.tsx

```diff
@@ -104,13 +112,21 @@ const selectEvents = createSelector(
   (eventIds, eventsById, day) =>
     eventIds
       .map((id) => eventsById[id])
-      .filter((event) => moment(event.startTime).isSame(day, 'day'))
+      .filter(
+        (event) =>
+          moment(event.startTime).isSame(day, 'day') ||
+          (moment(event.startTime).isSameOrBefore(day, 'day') &&
+            moment(event.endTime).isSameOrAfter(day, 'day') &&
+            moment.duration(
+              moment(event.endTime).diff(moment(event.startTime))
+            ) > moment.duration(1, 'days'))
+      )
```

Comment:
Is it possible to split this up a bit? Or at least add a comment explaining it :sweat_smile: